src/pygeoprocessing/geoprocessing.py

```diff
@@ -2591,16 +2604,15 @@ def convolve_2d(
             _LOGGING_PERIOD)
 
     LOGGER.info(
-        "convolution worker 100.0%% complete on %s",
-        os.path.basename(target_path))
+        f"convolution worker 100.0%% complete on "
```

Comment:
Does this double percent signify something or a typo?

Reply:
I learned something here... There are a lot of old strings that use `printf` style formatting, like `('percent complete %f%%' % count/100)`. You need that `%%` there, otherwise a % is interpreted as a replacement of something. I used this so much I assumed % always had to be escaped with another %, but that's only the case if you do the % replacement i.e. `'this is fine %'` `'but this crashes % %f' % 0.1`.  

Anyway, long way to say I removed that % and a couple others in afa1524663cac498b954923f33ed02b61d92389d

Reply:
Oh, also don't need the leading `f` there since nothing is being string replaced.